### DISCUSSION TITLE
Improve aliases for packages to not limit it to a single version.

### DISCUSSION THREAD

**Discussion by Sharcoux** [npm/rfcs#219] [RRFC] [answered]
Today, you can create an alias to a package using:

`npm i <alias>@npm:<package>`

Unfortunately, this will resolve into:

`"<alias>": "npm:<package>@<version>",`

The consequence of this is that your package will be fixed to a single version. There is no way to benefits from patches, or even accept all future versions. It should be possible to accept the following:

```json
"alias": "npm:package@latest"
"alias": "npm:package@^1.0.0"
"alias": "npm:package@>=1.0.0"
```

And so on.

**Comment by ruyadorno** [comment, marked as answer]
This already works as described in v7, just ran a test and confirmed it:

```sh
$ npm i foo@npm:abbrev@^1.0.0
$ npm ls foo
test@2.0.1 ~/test
└── foo@npm:abbrev@1.1.1
```
